participants/filecoin.net-disk.md

```diff
@@ -0,0 +1,56 @@
+# <Project Name>
+Filecoin.Net Disk
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+My project name is Filecoin.Net Disk
+My slack username is cannon, slack emil is cannon3758@outlook.com
+
+## (1) Brief Project Description
+This project is provide data store for users, which is use web UI interface to operation.
+And then we use lotus for data storage and retrieve deal from filecoin blockchain 
+
+
+## (2) Link to Project UI
+http://183.60.252.190:3334
+
+## (3) What does your application/UI do?
+Users can storage or retrieve data from filecoin network by this application 
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+My data is  sorts of video , we have the rights to the data for my plan. We have the data license available online. The data does'nt contain any  sensitive or illegal information 
```

Comment:
Great to hear! For our understanding, what kind of video is it? Are users restricted to just uploading video files or other files are also ok?

Reply:
Now is not restricted to video, all type of file is allow to upload